{"uuid": "881af475-9d63-45fd-86b2-5145970f3d80", "children": ["b3c2205b-c9e7-41b6-811c-134d56f95d86", "43f5658c-2a29-4db8-8f12-6be9747e2dda", "41144b1d-dcfe-41e1-96d1-a9abf1747938", "ad4718ac-17e8-4827-a652-9d5ad1502bb7"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586841396594, "stop": 1586841396594}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586841397643, "stop": 1586841409760}], "start": 1586841396594, "stop": 1586841409761}